### DISCUSSION TITLE
quick idea about comment

### DISCUSSION THREAD

**Discussion by HaveF** [rclement/datasette-dashboards#205] [Ideas]
I read [the news about datasette](https://datasette.substack.com/i/139440253/datasette-comments) and find a new plugin, [datasette-comments](https://github.com/datasette/datasette-comments).

When using the dashboard, we can also add comments to small widgets to mark which data or charts may be problematic.

It's just a very quick and rough idea.

**Comment by rclement** [comment]
Thanks @HaveF for the suggestion! I have seen the news too, this is very interesting.

However, reading through it quickly it seems the implementation is very table-centric for now and cannot easily be applied to other places such as dashboards and charts. Maybe that's something to be suggested in the upstream package (being able to comment anywhere).

I might be wrong, please correct me if do!

**Reply by HaveF** [reply]
@rclement you are right. I make [a suggestion in the upstream package](https://github.com/datasette/datasette-comments/discussions/16)
